widgetastic_manageiq/__init__.py

```diff
@@ -1022,6 +1022,49 @@ def workaround_save_issue(self):
         )
 
 
+class ReactCodemirror(Widget):
+    """Looks the same as ScriptBox but it changede in 5.11
+    https://github.com/JedWatson/react-codemirror"""
+
+    def __init__(self, parent, locator=None, read_locator=None, logger=None):
+        Widget.__init__(self, parent, logger=logger)
+        self.locator = locator
+        self.parent_locator = f"{locator}/.."  # we need to change visibility of this one
+        self.click_locator = f"{self.parent_locator}/.."  # the widget has very nested divs
+        self.read_locator = read_locator
+
+    def __locator__(self):
```

Comment:
I would set `ROOT` which will auto resolve locator.

Reply:
thanks, I'll do that